src/molecules/product-list-item/ProductListItem.html

```diff
@@ -0,0 +1,128 @@
+<a-list-item tag="div" class="a-product-list-item">
+  <div class="a-product-list-item__image">
+    <!--@slot Custom product image (Scoped slot)-->
+    <slot
+      name="image"
+      :alt="name"
+      :image-url="imageUrl"
```

Comment:
@e have to keep the same approach. If here we have `image-url` for  `name` slot we shoud also have `name` and `url` without prefixes